autogpt/commands/web_playwright.py

```diff
@@ -10,6 +10,7 @@
 from bs4 import BeautifulSoup
 
 from autogpt.processing.html import extract_hyperlinks, format_hyperlinks
+from autogpt.url_utils.validators import validate_url
```

Comment:
I did not found the decorator being used within playwright.py

Reply:
playwright isn't used on it's own so the url has already been validated

Reply:
Does `validate_url` need to be imported though, if it is not used in the file?

Reply:
Oh I misunderstood, definitely not it can be removed